src/reducers/reducer.js

```diff
@@ -230,6 +230,13 @@ const initialState = {
   networkSettings: PRE_SETTINGS_MAINNET,
   networkSettingsInvalid: {},
   networkSettingsStatus: NETWORKSETTINGS_STATUS.READY,
+  nanoContract: {
+    /**
+     * @property {boolean}
+     * @description Reflects Nano Contract's unleash feature toggle value.
+     */
+    available: false,
```

Comment:
We already have a redux state `featureToggles` with all the feature flags updated. Is this the same value? If it is, I prefer keeping only one place with feature flags updated, so we don't get confused when accessing and updating them.

Reply:
Sure!